src/ast.rs

```diff
@@ -12,55 +12,55 @@ impl<'a> fmt::Display for Name<'a> {
 
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub enum BinOp {
-    Plus,
-    Minus,
-    Times,
-    Divide,
-    Modulo,
+    Add,
+    Sub,
+    Mul,
+    Div,
+    Rem,
     And,
     Or,
     BitAnd,
     BitOr,
     BitXor,
     Shr,
     Shl,
-    Equal,
-    NotEqual,
-    LessThan,
-    GreaterThan,
-    LessThanOrEqual,
-    GreaterThanOrEqual,
+    Eq,
+    Neq,
+    Lt,
+    Gt,
+    Lte,
+    Gte,
 }
 
 impl<'a> fmt::Display for BinOp {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         use BinOp::*;
         match self {
-            Plus => write!(f, "+"),
-            Minus => write!(f, "-"),
-            Times => write!(f, "*"),
-            Divide => write!(f, "/"),
-            Modulo => write!(f, "%"),
+            Add => write!(f, "+"),
+            Sub => write!(f, "-"),
+            Mul => write!(f, "*"),
+            Div=> write!(f, "/"),
+            Rem => write!(f, "%"),
             And => write!(f, "&&"),
             Or => write!(f, "||"),
             BitAnd => write!(f, "&"),
             BitOr => write!(f, "|"),
             BitXor => write!(f, "^"),
             Shr => write!(f, ">>"),
             Shl => write!(f, "<<"),
-            Equal => write!(f, "=="),
-            NotEqual => write!(f, "!="),
-            LessThan => write!(f, "<"),
-            GreaterThan => write!(f, ">"),
-            LessThanOrEqual => write!(f, "<="),
-            GreaterThanOrEqual => write!(f, ">="),
+            Eq => write!(f, "=="),
+            Neq => write!(f, "!="),
+            Lt => write!(f, "<"),
+            Gt => write!(f, ">"),
+            Lte => write!(f, "<="),
+            Gte => write!(f, ">="),
         }
     }
 }
 
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub enum UnOp {
-    Minus,
+    Sub,
```

Comment:
Maybe Neg (short for negate) is a better choice?

Reply:
Sounds good to me: https://doc.rust-lang.org/std/ops/trait.Neg.html